File: temp/문자열 내림차순.java
import java.util.*;
import java.util.stream.Stream;


public String getDescSorting(String s) {
    String result = "";
    String strList [] = new String[s.length()];

    for (int i = 0; i < s.length(); i++) {
        strList[i] = String.valueOf(s.charAt(i));
    }

    Arrays.sort(strList, Comparator.reverseOrder());
    result = Arrays.stream(strList).reduce("", (acc, item) -> acc + item);

    return result;
}

File: temp/문자열 내림차순.js
function fn_getString(string){
    const charList = [];

    for (const char of string) {
        charList.push(char);
    }

    return charList
        .sort((a, b) => a > b ? -1 : 1)
        .reduce((acc, item) => acc += item, '');
}